From ad86f9f065b6799b45793c30e03747abd97d2833 Mon Sep 17 00:00:00 2001
From: Wenlin Kang <wenlin.kang@windriver.com>
Date: Fri, 3 Apr 2020 15:31:10 +0000
Subject: [PATCH] iio: adc: imx8qxp_adc: fix build error.

iio_info.driver_module has been droped with commit 97623c0a80a6(iio: drop
iio_info.driver_module and iio_trigger_ops.owner.), so here should remove
it from imx8qxp_adc_iio_info too.

This fixes:
drivers/iio/adc/imx8qxp_adc.c:501:3: error: 'const struct iio_in
   .driver_module = THIS_MODULE,
    ^~~~~~~~~~~~~

Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/iio/adc/imx8qxp_adc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/iio/adc/imx8qxp_adc.c b/drivers/iio/adc/imx8qxp_adc.c
index 7408fce6cdd8..a6a9c42d08cd 100644
--- a/drivers/iio/adc/imx8qxp_adc.c
+++ b/drivers/iio/adc/imx8qxp_adc.c
@@ -498,7 +498,6 @@ static int imx8qxp_adc_reg_access(struct iio_dev *indio_dev,
 }
 
 static const struct iio_info imx8qxp_adc_iio_info = {
-	.driver_module = THIS_MODULE,
 	.read_raw = &imx8qxp_adc_read_raw,
 	.debugfs_reg_access = &imx8qxp_adc_reg_access,
 };
-- 
2.17.1

